subprojects/diagnostics/src/integTest/groovy/org/gradle/api/tasks/diagnostics/DependencyInsightReportTaskIntegrationTest.groovy

```diff
@@ -3157,6 +3157,56 @@ planet:pluto:1.0.0
 """
     }
 
+    @ToBeFixedForInstantExecution(because = ":dependencyInsight")
+    def "reports a strictly on a transitive"() {
+        given:
+        def foo = mavenRepo.module("org", "foo", "1.2").publish()
+        mavenRepo.module("org", "foo", "1.5").publish()
+        mavenRepo.module("org", "bar", "1.0").dependsOn(foo).publish()
```

Comment:
:o: no need to fix right now but in the future, i'd avoid `foo` as artifactId and `foo` as variable name in the same test, especially given we have two versions of foo (and it is relevant which foo we depend on for the testcase)

Reply:
That was a mistake, I usually have the version in that variable name. Fixed!